promql/engine.go

```diff
@@ -659,7 +659,7 @@ func subqueryTimes(path []parser.Node) (time.Duration, time.Duration, *int64) {
 	return subqOffset, subqRange, tsp
 }
 
-func (ng *Engine) findMinMaxTime(s *parser.EvalStmt) (int64, int64) {
+func (ng *Engine) FindMinMaxTime(s *parser.EvalStmt) (int64, int64) {
```

Comment:
Now that it is exported, a comment would be nice.

Reply:
not being used anymore, removed this change